sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -12,6 +12,7 @@
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
+- Added support for failover databases and elastic pools.
```

Comment:
Sorry, looks like 1.32 was just published. Can you wipe out the 1.32 release notes and bump to 1.33?